Sourcery/Parsing/Parser.swift

```diff
@@ -453,18 +456,57 @@ extension Parser {
 
         return annotations
     }
+    
+    fileprivate func parseTypealiases(from source: [String: SourceKitRepresentable], containingType: Type?, processed: [[String: SourceKitRepresentable]]) -> [Typealias] {
+        var contents = self.contents
+        
+        //replace all processed substructures with whitespaces so that we don't process their typealiases again
+        for substructure in processed {
```

Comment:
if we are doing that, shouldn't we clear processed then? won't you repeat the same clearing out logic every time and for data that was already replaced?